cg/clients/freshdesk/models.py

```diff
@@ -20,17 +21,19 @@ class TicketCreate(BaseModel):
     type: str | None = None
 
 
-class TicketResponse(BaseModel):
-    """Freshdesk ticket response."""
+class Attachment(BaseModel):
+    id: int
+    name: str
+    content_type: str
+    size: int
+    created_at: str
 
-    attachments: list[dict[str, str]] = []
-    created_at: datetime | None = None
-    email: str
+
+class TicketResponse(BaseModel):
     id: int
-    name: str | None = None
-    priority: int
-    source: int
-    status: int
+    description: str
     subject: str
-    tags: list[str] = []
-    type: str | None = None
+    to_emails: Optional[List[str]] = None
+    status: int
+    priority: int
+    attachments: Optional[List[Attachment]] = None
```

Comment:
It seems like this is not used anywhere and can potentially be removed (unless you plan on using it in future PR:s!)

Reply:
I deleted it for now